resources/user-agents/crawlers/general-crawlers/general-crawlers.json

```diff
@@ -2328,6 +2328,54 @@
         {
           "match":  "NinjBot/*",
           "browser":  "ninj-bot"
+        },
+        {
+          "match":  "Yewno-Open",
```

Comment:
Please add a `*` at the end of all new rules, even if they are static. Otherwise they may are missing in the generated file

Reply:
Added